docs/getting-started/integrations-cdn/react-default-cdn.md

```diff
@@ -55,8 +287,8 @@ The `<CKEditor>` component supports the following properties:
 * `onBlur` &ndash; A function called when the editor was blurred. See the {@link module:engine/view/document~Document#event:blur `editor.editing.view.document#blur`} event.
 * `onFocus` &ndash; A function called when the editor was focused. See the {@link module:engine/view/document~Document#event:focus `editor.editing.view.document#focus`} event.
 * `onError` &ndash; A function called when the editor has crashed during the initialization or during the runtime. It receives two arguments: the error instance and the error details. Error details is an object that contains two properties:
-  * `{String} phase`: `'initialization'|'runtime'` &ndash; Informs when the error has occurred (during the editor or context initialization, or after the initialization).
-  * `{Boolean} willEditorRestart` &ndash; When `true`, it means that the editor component will restart itself.
+	* `{String} phase`: `'initialization'|'runtime'` &ndash; Informs when the error has occurred (during the editor or context initialization, or after the initialization).
+	* `{Boolean} willEditorRestart` &ndash; When `true`, it means that the editor component will restart itself.
```

Comment:
The info about types is useful but it's inconsistent with the bullet points above. Maybe we should remove it?
```suggestion
	* `phase`: `'initialization'|'runtime'` &ndash; Informs when the error has occurred (during the editor or context initialization, or after the initialization).
	* `willEditorRestart` &ndash; When `true`, it means that the editor component will restart itself.
```